src/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -979,6 +1002,10 @@ private void processNewState(int oldState, int state)
 			mTimeline.setShuffleMode(shuffleMode(state));
 		if ((toggled & MASK_FINISH) != 0)
 			mTimeline.setFinishAction(finishAction(state));
+
+		if((toggled & FLAG_DUCKING) != 0) {
+			mMediaPlayer.setIsDucking((state & FLAG_DUCKING) != 0);
+		}
```

Comment:
This will probably fail if the media-player gets swapped during ducking.
Adding
mPreparedMediaPlayer.setIsDucking((state & FLAG_DUCKING) != 0);
should solve this.